go/mysql/hex/hex_test.go

```diff
@@ -98,5 +98,5 @@ func TestDecodeBytes(t *testing.T) {
 	// DecodeBytes should return an error for "é" as
 	// hex.decode returns an error for non-ASCII characters
 	err = DecodeBytes([]byte("testDst"), []byte("é"))
-	assert.Error(t, err)
+	assert.Error(t, err, "DecodeBytes() should have errored as src contains non-ASCII character")
```

Comment:
Any reason for having a custom message at all?

Reply:
We have added custom messages earlier in other tests too. So, I added here.

Reply:
I don’t think it add a lot of value here? I think we can go for the minimal setup and only add something custom if there’s explicit value in doing so.

Reply:
Done.